packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import * as KeyCode from 'keycode-js';
+import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
+import styles from './WorklistDataGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String representing an attribute which identifies the element (or elements) that labels the grid.
+   */
+  ariaLabelledby: PropTypes.string,
+  /**
+   * String that labels the grid for accessibility.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * String that will be used to identify the Grid. This value will be used as the id attribute of the overall grid container,
+   * and it will be used to prefix other id attributes used for internal componentry.
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * Data for column headers. Columns will be presented in the order given.
+   */
+  columnHeaders: WorklistDataGridPropTypes.columnHeaderShape,
+  /**
+   * Data for content in the body of the Grid. Rows will be rendered in the order given.
+   */
+  rows: PropTypes.arrayOf(WorklistDataGridPropTypes.rowShape),
+  /**
+   * String that specifies the default column width. Any valid CSS width value is accepted.
+   * This value will be used if no overriding width value is provided on a per-column basis.
+   */
+  columnHeaderDefaultWidth: PropTypes.string,
+  /**
+   * String that specifies the default row height. Any valid CSS width value is accepted.
+   * This value will be used if no overriding height value is provided on a per-row basis.
+   */
+  rowDefaultHeight: PropTypes.string,
```

Comment:
This should probably just be rowHeight.  The framework would determine this value and pass the appropriate height.

Reply:
Since each row could potentially have a different height, we provided this prop to make it easier for the consumer to specify only 1 height and override it for row(s) requiring a different height. Similar to columnWidth, we changed the prop name to ‘rowHeight’.